### DISCUSSION TITLE
Obsidian 个人插件开发纪实——0x01

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#17] [General]
# Obsidian 个人插件开发纪实——0x01

Obsidian 插件开发纪实会是一个系列的文章，主要是记录我真正从零开始开发 Obsidian 插件做为副业项目的过程，这中间会涉及到 js、nodejs、css、React 等技术

https://www.edony.ink/personal-obsidian-plugin-development-reocord-1/

**Comment by 2Circle** [comment]
Obsidian使我们相遇,很不错的blog!!!

**Reply by edonyzpc** [reply]
谢谢🙏，Obsidian 最近升级了 1.0，后面会更新一下这个系列
